components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -263,6 +264,7 @@ public class OAuth2AuthzEndpoint {
     private static final String SET_COOKIE_HEADER = "Set-Cookie";
     private static final String REGEX_PATTERN = "regexp";
     private static final String OIDC_SESSION_ID = "OIDC_SESSION_ID";
+    public static final String DYNAMIC_TOKEN_DATA_FUNCTION = "dynamicTokenData = function (";
```

Comment:
You are using this constant in the same class. So make this private. And given that we need to anyway move out from current behavior we should not introduce public constants like these

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2437/commits/6ec88a197a4a6608bbc8a1e69f6c53c085d4f54f